alert_test.go

```diff
@@ -41,25 +44,18 @@ func TestAlerts_PaginatedFind(t *testing.T) {
 		t.Fatal(err)
 	}
 	invoked := ((a.client).(*MockAlertClient)).InvokedCount
-	if invoked != 2 {
-		t.Errorf("paginated search, expected 2, got %d", invoked)
-	}
-
-	if alerts[0].Name != "Excessive consumption of inodes" {
-		t.Errorf("alert name incorrect: %s", alerts[0].Name)
-	}
-
-	if len(alerts[0].Tags) != 2 {
-		t.Errorf("alert tags, expected 2, got %d", len(alerts[0].Tags))
-	}
+	assert.Equal(2, invoked)
```

Comment:
did the lint complain about this?  Wouldn't ErrorF be more efficient in listing all the failures at once?

Reply:
No, the lint error didn't call this one out explicitly.  I could have left this as-is, but I changed it for the sake of readability. 
 
Keep in mind that assert.Equal() also list all the failures at once.  That is, if assert.Equal() fails, the test continues to run.

assert.Equal() is part of Testify which is a unit testing framework for GO.  It essentially does what t.Errorf does but it is more compact and readable.  if the two values aren't equal, it prints the out the expected value, the actual value, and the line number where the assert failed.  assert.Equal() can also compare slices and structs

Reply:
sounds good